caldav/elements.go

```diff
@@ -230,8 +248,11 @@ func (r *reportReq) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error {
 }
 
 type mkcolReq struct {
-	XMLName      xml.Name              `xml:"DAV: mkcol"`
-	ResourceType internal.ResourceType `xml:"set>prop>resourcetype"`
-	DisplayName  string                `xml:"set>prop>displayname"`
-	// TODO this could theoretically contain all addressbook properties?
+	XMLName                       xml.Name                      `xml:"DAV: mkcol"`
+	ResourceType                  internal.ResourceType         `xml:"set>prop>resourcetype"`
+	DisplayName                   string                        `xml:"set>prop>displayname"`
+	Description                   string                        `xml:"set>prop>calendar-description"`
+	CalendarColor                 string                        `xml:"set>prop>calendar-color"`
```

Comment:
We should probably specify the `http://apple.com/ns/ical/` namespace for this prop?